internal/sqladapter/testing/adapter.go.tpl

```diff
@@ -117,7 +117,7 @@ func TestInsertReturning(t *testing.T) {
 	assert.NotZero(t, itemMap["id"], "Must not be zero after inserting")
 
 	itemStruct := struct {
-		ID int `db:"id,omitempty"`
```

Comment:
Does this file need to have .tpl suffix? Would be nice to see Go formatting by default.